packages/xml-views-validation/test/validators/element/unknown-tag-name-spec.ts

```diff
@@ -503,6 +515,53 @@ describe("the unknown tag name validation", () => {
             </mvc:View>`
           );
         });
+
+        it("will not detect an issue for sap.ui.core.ExtensionPoint as top level element in sap.ui.core.mvc.View", () => {
+          assertNoIssues(
+            `<mvc:View xmlns:mvc="sap.ui.core.mvc" xmlns:core="sap.ui.core">
+                <core:ExtensionPoint name="extension1"/>
+            </mvc:View>`
+          );
+        });
+
+        it("will not detect an issue for sap.ui.core.ExtensionPoint as top level element in sap.ui.core.Fragment", () => {
+          assertNoIssues(
+            `<FragmentDefinition xmlns="sap.ui.core">
+                <ExtensionPoint name="extension1"/>
+            </FragmentDefinition>`
+          );
+        });
+
+        it("will not detect an issue for sap.ui.core.ExtensionPoint as nested element in sap.ui.core.mvc.View", () => {
+          assertNoIssues(
+            `<mvc:View xmlns:mvc="sap.ui.core.mvc" xmlns:core="sap.ui.core" xmlns:m="sap.m">
+                <m:Page>
+                  <m:content>
+                    <core:ExtensionPoint name="extension1"/>
+                  </m:content>
+                </m:Page>
+            </mvc:View>`
+          );
+        });
+
+        it("will not detect an issue for sap.ui.core.ExtensionPoint as nested element in sap.ui.core.Fragment", () => {
+          assertNoIssues(
+            `<FragmentDefinition xmlns="sap.ui.core">
+              <m:Panel>
+                <m:content>
+                  <core:ExtensionPoint name="extension1"/>
+                </m:content>
+              </m:Panel>
+            </FragmentDefinition>`
+          );
+        });
+
+        it("will detect an issue for sap.ui.core.ExtensionPoint in the root tag", () => {
+          assertSingleIssue(
```

Comment:
This test is failing. Could you give me a hint, please? Why there is no error logged?

Reply:
I have no idea just from looking at the code.
I will have to debug this, but this will only happen next week

Reply:
Hello @flovogt 

So the test infrastructure assumes you will mark the text areas (ranges) where issues have been detected.
These markers are applied directly on the input snippet, e.g:
- `            <🢂m:Button_TYPO🢀>`

```typescript
 it("will detect an invalid class name under class that has default aggregation", () => {
        assertSingleIssue(
          `<mvc:View
            xmlns:mvc="sap.ui.core.mvc"
            xmlns:m="sap.m">
            <🢂m:Button_TYPO🢀>
            </m:Button_TYPO>
          </mvc:View>`,
          buildMessage(UNKNOWN_CLASS_IN_NS.msg, "Button_TYPO", "sap.m")
        );
      });

Use **copy/paste** to copy these special unicode arrow characters...
```

Reply:
Hi @bd82 ,
thanks a lot for the hint. Test is adjusted.
Best Regards,
Florian